pages/linux/perf.md

```diff
@@ -0,0 +1,19 @@
+# perf
+
+> Framework for linux performance counter measurements.
+
+- Display basic performance counter stats for a command `gcc -c -O3 hello.c`:
+
+`perf stat {{gcc -c -O3 hello.c}}`
```

Comment:
Please add a - after stat. Also, would you like to remove the -c and -O3 flags to keep it simple ?

Reply:
sure, I would.
I can't add `-` after stat, as it works without it, whatever you write after `perf stat` is taken as an executable.
Also adding a `-` throws an error as the program is not written to handle it.

Reply:
Ok, I must have been looking at older documentation.